app/src/test/java/com/fstyle/structure_android/ExampleUnitTest.java

```diff
@@ -2,7 +2,7 @@
 
 import org.junit.Test;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
```

Comment:
ERROR: Wrong order for org.junit.Assert.assertEquals import. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports